hibernate-core/src/main/java/org/hibernate/metamodel/mapping/internal/BasicValuedSingularAttributeMapping.java

```diff
@@ -73,13 +78,21 @@ public BasicValuedSingularAttributeMapping(
 		this.navigableRole = navigableRole;
 		this.tableExpression = tableExpression;
 		this.mappedColumnExpression = mappedColumnExpression;
+		this.isMappedColumnExpressionFormula = isMappedColumnExpressionFormula;
 		this.valueConverter = valueConverter;
 		this.jdbcMapping = jdbcMapping;
 
 		if ( valueConverter == null ) {
 			domainTypeDescriptor = jdbcMapping.getJavaTypeDescriptor();
 		}
 		else {
+			if ( isMappedColumnExpressionFormula ) {
```

Comment:
Why disallow this?  In theory it should be fine.

Reply:
I forgot what was in my mind when I coded this a long time ago, :). I deleted this as you suggested. Thanks.